tests/basic/storage_pool_test.go

```diff
@@ -5188,3 +5190,179 @@ var _ = Describe("{ResizePoolDrivesInDifferentSize}", func() {
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("VolDeletePoolExpand", "Delete volume which has ~200G data and do an expansion of pool by resize", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	var newContexts []*scheduler.Context
+	stepLog := "should get the existing storage node and write ~200G data to a volume"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("voldeletepoolexpand-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		log.Infof("Need to check if volume is close to 200G occupied")
+		vol, err := getVolumeWithMinimumSize(contexts, 90)
+		// We will change the size, after modifying/deploying a vdbench/fio to write ~200G. Current vdbench is writing 98G
+		dash.VerifyFatal(err, nil, "Checking if the desired volume is obtained ")
+		volID := vol.ID
+		volName := vol.Name
+		log.Infof("The volume that is having size used around 190 G is %s with name %s", volID, volName)
+		stepLog = "Attempt to expand the pool by resize operation"
+		var poolIDToResize string
+		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+		dash.VerifyFatal(len(pools) > 0, true, " Storage pools exist?")
+		// pick a pool from a pools list and resize it
+		appVol, err := Inst().V.InspectVolume(volID)
+		dash.VerifyFatal(err, nil, "Checking if the Volume inspect is success for the desired volume")
+		// Get the pool UUID on which the volume which is ~190G exist
+		poolIDToResize = appVol.ReplicaSets[0].PoolUuids[0]
+		log.FailOnError(err, "error identifying pool to run test")
+		dash.VerifyFatal(len(poolIDToResize) > 0, true, fmt.Sprintf("Expected poolIDToResize to not be empty, pool id to resize %s", poolIDToResize))
+		poolToBeResized := pools[poolIDToResize]
+		dash.VerifyFatal(poolToBeResized != nil, true, "Pool to be resized exist?")
+		// px will put a new request in a queue, but in this case we can't calculate the expected size,
+		// so need to wait until the ongoing operation is completed
+		time.Sleep(time.Second * 60)
+		stepLog = "Verify that pool resize is not in progress"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			if poolResizeIsInProgress(poolToBeResized) {
+				// wait until resize is completed and get the updated pool again
+				poolToBeResized, err = GetStoragePoolByUUID(poolIDToResize)
+				log.FailOnError(err, "Failed to get pool using UUID ")
+			}
+		})
+		stepLog = "set pool label, before pool expand"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			poolLabelToUpdate := make(map[string]string)
+			poolLabelToUpdate["cust-type"] = "test-label"
+			storageNode, err := GetNodeWithGivenPoolID(poolIDToResize)
+			log.FailOnError(err, "Failed to get pool using UUID ")
```

Comment:
nit : extra space , please update where ever it is necessary

similar to above comments , there are few places where we have similar nits ,please take care of that

Reply:
and add pool UUID to the message

Reply:
Resolved